### DISCUSSION TITLE
Inclusion in the official Debian package source

### DISCUSSION THREAD

**Discussion by linux-lukas** [VSCodium/vscodium#1025] [General]
Have you ever thought about contacting the Debian team to check whether and to what extent it would be possible to be included in the official package sources of the Debian project and thus also directly in all package sources based on Debian? #

For example, distros based on Debian, such as Ubuntu, could be made less dependent on Snap.

**Comment by paulcarroty** [comment]
No sense to contact a million distros and request the package. It's up to them to handle their software.

**Comment by Tachi107** [comment]
I agree with @paulcarroty, and there are also some technical issues that prevent Electron apps from being packaged by Debian. 

In fact, somebody already asked Debian developers to package vscode, and [the ticket](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=898259) is still open, but it would require packaging of Electron, and that's a really big task that nobody has completed yet.
